packages/client-api-schema/src/__tests__/good_sample_requests.ts

```diff
@@ -95,8 +95,14 @@ export const createChannel = {
       {
         token: '0x0',
         allocationItems: [
-          {destination: '0x63e3fb11830c01ac7c9c64091c14bb6cbaac9ac7', amount: '0x06f05b59d3b20000'},
-          {destination: '0x63e3fb11830c01ac7c9c64091c14bb6cbaac9ac7', amount: '0x06f05b59d3b20000'}
+          {
+            destination: '0x63e3fb11830c01ac7c9c64091c14bb6cbaac9ac7',
+            amount: '0x00000000000000000000000000000000000000000000000006f05b59d3b20000'
```

Comment:
Why does this need to be padded?

Reply:
In order to pass the schema after the latter changed (merit of which we are discussing in other threads on this page).